packages/backend/src/helpers/timestampLoader.mjs

```diff
@@ -0,0 +1,100 @@
+import fetch from 'node-fetch'
+import { ETH_PROVIDER_URL, provider } from '../config.mjs'
+
+export class TimestampLoader {
+  constructor(db) {
+    this.db = db
+  }
+
+  start() {
+    this._start()
+  }
+
+  async _start() {
+    for (;;) {
```

Comment:
what does this mean?

I will need to read this 4 more times.. but UI is showing timestamps everywhere they were working before 👍

Reply:
It's an infinite loop. The middle expression defaults to `true` so this could also be written like `for (; true ;)` or `for (let x = 0; true; x++)`